build_tools/third_party/mlir-hlo/CMakeLists.txt

```diff
@@ -19,9 +19,9 @@ external_cc_library(
   ROOT
     ${TF_MLIR_HLO_SOURCE_DIR}
   INCLUDES
-    "${TF_MLIR_HLO_SOURCE_DIR}/"
     "${TF_MLIR_HLO_SOURCE_DIR}/include/"
-    "${TF_MLIR_HLO_BINARY_DIR}/"
+    "${TF_MLIR_HLO_SOURCE_DIR}/stablehlo/"
```

Comment:
```
Analyzing: 2664 targets (53 packages loaded, 1333 targets configured)
ERROR: /home/runner/.cache/bazel/_bazel_runner/ff3f6aeb270cfbbde130ed94a6fa7853/external/mlir-hlo/BUILD:22:11: no such package '@mlir-hlo//third_party/tensorflow/compiler/xla/mlir_hlo/stablehlo': BUILD file not found in directory 'third_party/tensorflow/compiler/xla/mlir_hlo/stablehlo' of external repository @mlir-hlo. Add a BUILD file to a directory to mark it as a package. and referenced by '@mlir-hlo//:hlo_ops_td_files'
ERROR: /home/runner/.cache/bazel/_bazel_runner/ff3f6aeb270cfbbde130ed94a6fa7853/external/mlir-hlo/BUILD:442:11: no such package '@mlir-hlo//third_party/tensorflow/compiler/xla/mlir_hlo/stablehlo': BUILD file not found in directory 'third_party/tensorflow/compiler/xla/mlir_hlo/stablehlo' of external repository @mlir-hlo. Add a BUILD file to a directory to mark it as a package. and referenced by '@mlir-hlo//:mlir_hlo'
```

Reply:
Yeah. A known issue. mlir-hlo's BUILD is broken; @burmako is fixing it.

Reply:
Done by https://github.com/tensorflow/mlir-hlo/commit/3b1b023db925e1da4c6368ee379889c8b7074a53.